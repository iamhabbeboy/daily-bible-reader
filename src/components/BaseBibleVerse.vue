<template>
  <div
    class=" px-6 my-20 mx-auto p-10 rounded-md 
    bg-gray-700 w-8/12 bg-opacity-30 shadow-md "
  >
    <Header />
    <Content />
    <div class="flex mt-5">
      <Pagination />
      <Option />
    </div>
  </div>
</template>
<script>
import Header from "@/components/verse/Header";
import Content from "@/components/verse/Content";
import Pagination from "@/components/verse/Pagination";
import Option from "@/components/verse/Option";

export default {
  components: {
    Header,
    Content,
    Pagination,
    Option
  }
};
</script>
