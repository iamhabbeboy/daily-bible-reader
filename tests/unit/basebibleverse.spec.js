import { shallowMount } from "@vue/test-utils";
import BaseBibleVerse from "@/components/BaseBibleVerse";

describe("BaseBibleVerse.vue", () => {
  it("take snapshot", () => {
    const wrapper = shallowMount(BaseBibleVerse);
    expect(wrapper).toMatchSnapshot();
  });
});
