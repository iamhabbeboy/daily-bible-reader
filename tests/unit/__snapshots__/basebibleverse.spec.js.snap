// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseBibleVerse.vue renders props.msg when passed 1`] = `
<div class=" px-6 my-20 mx-auto p-10 rounded-md 
  bg-gray-700 w-8/12 bg-opacity-30 shadow-md ">
  <header-stub></header-stub>
  <content-stub></content-stub>
  <div class="flex mt-5">
    <pagination-stub></pagination-stub>
    <option-stub></option-stub>
  </div>
</div>
`;

exports[`BaseBibleVerse.vue take snapshot 1`] = `
<div class=" px-6 my-20 mx-auto p-10 rounded-md 
  bg-gray-700 w-8/12 bg-opacity-30 shadow-md ">
  <header-stub></header-stub>
  <content-stub></content-stub>
  <div class="flex mt-5">
    <pagination-stub></pagination-stub>
    <option-stub></option-stub>
  </div>
</div>
`;
